coalib/parsing/filters/CanDetectFilter.py

```diff
@@ -0,0 +1,31 @@
+def can_detect_filter(bears, detects):
+    if detects:
+        bears = filter_bears_by_can_detects(bears, detects)
+    return bears
+
+
+def filter_bears_by_can_detects(bears, detects):
+    local_bears, global_bears = bears
+    local_bears = filter_section_bears_by_can_detects(local_bears, detects)
+    global_bears = filter_section_bears_by_can_detects(
```

Comment:
@Makman2  I've added implementation with class. Now, to add new filter, new class has to be derived from `FilterBase` and implement an abstract method.